Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -310,6 +312,10 @@ void AzAssetBrowserWindow::RegisterViewClass()
     AzToolsFramework::ViewPaneOptions options;
     options.preferedDockingArea = Qt::LeftDockWidgetArea;
     AzToolsFramework::RegisterViewPane<AzAssetBrowserWindow>(LyViewPane::AssetBrowser, LyViewPane::CategoryTools, options);
+
+    options.showInMenu = false;
+    const QString name = QString("%1 2").arg(LyViewPane::AssetBrowser);
+    AzToolsFramework::RegisterViewPane<AzAssetBrowserWindow>(qPrintable(name), LyViewPane::CategoryTools, options);
```

Comment:
Does the design intend to only have 2 asset browsers, or is the user going to be able to open any number of asset browsers? If the latter, we wouldn't need to register an additional one it could just be registered as a multi-view pane?

Reply:
Cool makes sense.

My only other comment is that in the mockups the 2 is surrounded by parentheses so it would be `Asset Browser (2)`, did that get changed in an offline discussion or should that be updated here?

Reply:
Thanks! Looks good, a new AR build is in progress.